packages/terralib/lua/Layer.lua

```diff
@@ -655,6 +655,15 @@ Layer_ = {
 		end
 		
 		return luaPropNames
+	end,
+	--- Returns the dummy value of a raster layer. If the layer does not have a raster representation
+	-- then it returns a nil value . The bands of the raster layer are named from zero to the number of
+	-- bands minus one, if the band is greater than that, it returns an error.
+	-- @arg band The band number
```

Comment:
There should exist a point after `band number`. `-doc`  should fail in this case. If not, please tell me.